docs/deploying_apps/deploying_node_js.md

```diff
@@ -19,8 +19,6 @@ These steps assume you have already carried out the setup process explained in t
 
 3. Include an npm ``package.json`` file to specify dependencies. The file should also specify a `start` command used to launch the app.
   
-    The presence of this file is essential to have the application detected as Node.js.
```

Comment:
I agree that it requires it and the script is called `detect.yml`, but not to detect which buildpack to use. We've already told the user to add this to `manifest.yml`

```
  buildpack: nodejs_buildpack
```

I'll reword this to say the buildpack always needs it.